cmd/plugin/rpaasv2/cmd/info.go

```diff
@@ -0,0 +1,226 @@
+	if autoscale == nil {
+		return ""
+	}
+	var buffer bytes.Buffer
+	table := tablewriter.NewWriter(&buffer)
+	table.SetHeader([]string{"Replicas", "Target Utilization"})
+	table.SetAutoWrapText(true)
+	table.SetRowLine(false)
+	var max, min, cpuPercentage, memPercentage string
+
+	if autoscale.MaxReplicas != nil {
+		max = fmt.Sprintf("Max: %s", strconv.Itoa(int(*autoscale.MaxReplicas)))
+	} else {
+		max = "Max: N/A"
+	}
+	if autoscale.MinReplicas != nil {
+		min = fmt.Sprintf("Min: %s", strconv.Itoa(int(*autoscale.MinReplicas)))
+	} else {
+		min = "Min: N/A"
+	}
+	if autoscale.CPU != nil {
+		cpuPercentage = fmt.Sprintf("CPU: %s%%", strconv.Itoa(int(*autoscale.CPU)))
+	} else {
+		cpuPercentage = "CPU: N/A"
+	}
+	if autoscale.Memory != nil {
+		memPercentage = fmt.Sprintf("Memory: %s%%", strconv.Itoa(int(*autoscale.Memory)))
+	} else {
+		memPercentage = "Memory: N/A"
+	}
+	data := [][]string{
+		{max, cpuPercentage},
+		{min, memPercentage},
+	}
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func writeAddressesOnTableFormat(adresses []clientTypes.InstanceAddress) string {
+	data := [][]string{}
+	for _, address := range adresses {
+		data = append(data, []string{address.Hostname, address.IP})
+	}
+	var buffer bytes.Buffer
+	table := tablewriter.NewWriter(&buffer)
+	table.SetHeader([]string{"Hostname", "IP"})
+	table.SetRowLine(true)
+	table.SetAutoWrapText(true)
+	table.SetColumnAlignment([]int{tablewriter.ALIGN_LEFT, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_LEFT})
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func writeInfoRoutesOnTableFormat(routes []clientTypes.Route) string {
+	data := [][]string{}
+	for _, route := range routes {
+		data = append(data, []string{route.Path, route.Destination})
+	}
+	var buffer bytes.Buffer
+	table := tablewriter.NewWriter(&buffer)
+	table.SetHeader([]string{"Path", "Destination"})
+	table.SetRowLine(true)
+	table.SetAutoWrapText(true)
+	table.SetColumnAlignment([]int{tablewriter.ALIGN_LEFT, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_LEFT})
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func formatTags(tags []string) string {
+	return strings.Join(tags, ", ")
+}
+
+func writeBindsOnTableFormat(binds []v1alpha1.Bind) string {
+	data := [][]string{}
+	for _, bind := range binds {
+		data = append(data, []string{bind.Name, bind.Host})
+	}
+	var buffer bytes.Buffer
+	table := tablewriter.NewWriter(&buffer)
+	table.SetHeader([]string{"App", "Address"})
+	table.SetRowLine(true)
+	table.SetAutoWrapText(true)
+	table.SetColumnAlignment([]int{tablewriter.ALIGN_LEFT, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_CENTER, tablewriter.ALIGN_LEFT})
+	table.AppendBulk(data)
+	table.Render()
+
+	return buffer.String()
+}
+
+func writeInfoOnJSONFormat(w io.Writer, payload *clientTypes.InstanceInfo) error {
+	message, err := json.Marshal(payload)
```

Comment:
Just to maintain the the standard of other functions like as `writeRoutesOnJSONFormat` and `writeBlocksOnJSONFormat`.

```suggestion
	message, err := json.MarshalIndent(payload, "", "\t")
```